### PR TITLE
[Keyboard] Add meow48

### PR DESCRIPTION
**Pull request by tsubuan145**
The operation has been confirmed.

## Types of Changes

- [x] Core
- [x] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/meow48/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,           KC_NO,
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,           KC_NO,
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,              KC_NO,              KC_NO,    KC_NO,    KC_NO,   KC_NO,  KC_NO
+    ),
+    
+     [7] = LAYOUT(
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,   KC_NO,  KC_NO,
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,           KC_NO,
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,           KC_NO,
+        KC_NO,  KC_NO,    KC_NO,    KC_NO,    KC_NO,              KC_NO,              KC_NO,    KC_NO,    KC_NO,   KC_NO,  KC_NO
```
Comment (waffle87) [inline keyboards/meow48/keymaps/via/keymap.c:85]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/meow48/keymaps/via/keymap.c (hunk 1 above)
Comment (tsubuan145) [reply keyboards/meow48/keymaps/via/keymap.c]: @waffle87 I fixed it,

**Comment by tsubuan145** [comment]
Here is the image of the keyboard.

**Comment by tsubuan145** [comment]
I fixed config.h because ProMicro generated a fever due to the power setting of the LED.

**Comment by tsubuan145** [comment]
Fixed a bug that the OLED was not displayed properly.

**Pull request closed without merging**